daemons/attrd/attrd_nodes.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 the Pacemaker project contributors
```

Comment:
Update the copyright date - I didn't check all the other files, but this one being new was obvious.

Reply:
This was all written last year